### ISSUE TITLE
[Feature Request] Ship with Typescript definition

### ISSUE THREAD

**Issue by someone1** [egoist/vue-timeago#76]
I added a definition file myself for now, but was hoping this could ship with one (one that's possibly more complete/accurate).

```typescript
declare module 'vue-timeago';
```

**Comment by egoist** [comment]
Yeah I'd love it if someone could publish one to definitelyTyped.

**Comment by alkaliphile** [comment]
@someone1 would you mind sharing that definition file please?

**Comment by someone1** [comment]
There really isn't much other than what I posted above - bare minimum to get the compiler to stop complaining - sorry!
